docs/cheaha/software/modules.md

```diff
@@ -52,6 +52,42 @@ If you want to revert to the default modules, you can use:
 module reset
 ```
 
+## Saving Modules using Collections
+
+To save time in typing in long list of modules everytime you work on a project, you can save the desired list of modules using module collection. To acheive this first load the desired modules, and save it using a collection name as below,
```

Comment:
At the end of the first sentence, please change to "using a module collection."

I avoid using "ordering" words in writing, as they add clutter but not much clarity. So, in the second sentence, "To achieve this, load the desired modules and save them to a collection, as shown below." This is a style choice, not required, but I think it flows well.

Reply:
Sure. Incorporated the changes.